Tests/Magick.NET.Tests/Shared/Pixels/SafePixelCollectionTests/TheToByteArrayMethod.cs

```diff
@@ -50,6 +51,21 @@ public void ShouldReturnPixelsWhenAreaIsCorrecy()
                 }
             }
 
+            [TestMethod]
+            public void ShouldReturnPixelsWhenAreaAndMappingAreCorrect()
```

Comment:
Maybe call this `ShouldReturnPixelsWhenAreaIsCorrectAndMappingIsEnum`? And use this nameing style in the tests for the other methods?